### PR TITLE
A drop method with shift click ability

### PR DESCRIPTION
**Pull request by ChrisBPB**
Reduces inventory drop time by over 15 seconds

### CODE REVIEW

File: src/main/java/org/powerbot/script/rt4/Inventory.java [hunk 1]
```diff
@@ -99,6 +99,21 @@ public Component component() {
...
+	
+	/**
+     	* Drops specified item via regular or shift dropping.
+     	*
+     	* @param i The item to drop
+     	* @param shift Shift dropping, if true the method will verify it is enabled and fall back to regular if not
+     	* @return Success
+     	*/
+    	public boolean drop(Item i, Boolean shift){
+		if(shift && (ctx.varpbits.varpbit(1055) & 131072)>0){
```
Comment (ArtificialPB) [inline src/main/java/org/powerbot/script/rt4/Inventory.java]: Here's a cleaner check to see if shift dropping is enabled:
```
public boolean shiftDropping() {
        return ctx.varpbits.varpbit(1055, 17, 0x1) == 1;
    }
```

**Comment by ChrisBPB** [comment]
have updated it and separated the check for potential other methods

**Comment by Strikegently** [comment]
1. Can you rename `shiftDropping` so it's more apparent what it does?
Maybe `shiftDroppingEnabled` or `canUseShiftDropping`

2. Is there a reason the `shift` parameter isn't primitive?

**Comment by ChrisBPB** [comment]
updated

somehow got into the habit of Boolean instead

**Pull request merged**